src/openapi_parser/builders/schema.py

```diff
@@ -111,7 +111,7 @@ def create(self, data: dict) -> Schema:
         try:
             schema_type = data['type']
         except KeyError:
-            raise ParserError("Schema does not contain 'type' property")
+            schema_type = DataType.ANY_OF
```

Comment:
Would be nice to have some some log entry about implicit type assignment, maybe warning with the previous exception text or similar?